README.md

```diff
@@ -2,7 +2,7 @@
 
 ----------------------------------------
 
-A RESTful API for querying osm planet dumps
+A RESTful API for querying and creating osm planet dumps
```

Comment:
the service does not create osm planet dumps it saves its metadata, I would rephrase that

Reply:
done.